pkg/inject/xray.go

```diff
@@ -119,6 +123,10 @@ func (m *xrayMutator) checkConfig() error {
 		missingConfig = append(missingConfig, "xRayDaemonPort")
 	}
 
+	if m.mutatorConfig.xRayLogLevel == "" {
```

Comment:
Just trying to understand why we need this check. In what scenarios can this happen. Rest looks good.